exports-sql-data.sh

```diff
@@ -0,0 +1,14 @@
+# Exports all databases data from a Cloud SQL instance to a Google Cloud Storage bucket as a SQL dump file
```

Comment:
Add sample command to execute this script

Reply:
OK

Reply:
any update on this?